### PR TITLE
Add missing raster cache LRU accounting to Get()

### PR DESCRIPTION
**Pull request by cbracken**
RasterCache::Get() methods were not updating the RasterCache::Entry
access_count and used_this_frame fields, as is done in
RasterCache::Prepare(). This can result in onscreen images being evicted
from the cache as new entries are created (e.g. as new elements scroll
onscreen).

### CODE REVIEW

**Comment by cbracken** [comment]
See https://github.com/flutter/flutter/issues/50226

File: flow/raster_cache.cc [hunk 1]
```diff
@@ -231,13 +231,25 @@ RasterCacheResult RasterCache::Get(const SkPicture& picture,
                                    const SkMatrix& ctm) const {
   PictureRasterCacheKey cache_key(picture.uniqueID(), ctm);
   auto it = picture_cache_.find(cache_key);
-  return it == picture_cache_.end() ? RasterCacheResult() : it->second.image;
+  if (it == picture_cache_.end()) {
+    return RasterCacheResult();
+  }
+  Entry& entry = const_cast<Entry&>(it->second);
```
Comment (chinmaygarde) [inline flow/raster_cache.cc]: Can we dry this up just a little bit by creating a method that takes a key and returns a cache result whose entry has its access counted bumped already?

**Comment by cbracken** [comment]
Fixes flutter/flutter#50226

**Comment by cbracken** [comment]
Closed in favour of #16434.

**Pull request closed without merging**